R/fetch.R

```diff
@@ -0,0 +1,339 @@
+# .fetch_word -------------------------------------------------------------
+
+
+#' Fetch the Content of a Wiktionary Word Page
+#'
+#' @param word The word to fetch.
+#'
+#' @return Character; the word's page, in wikitext format.
+#' @keywords internal
+.fetch_word <- function(word) {
+  content <- tryCatch( # nocov start
+    {
+      WikipediR::page_content(
```

Comment:
This is the only place we use WikipediR, right? I noticed you downgraded the version requirement. Maybe soon we can remove that dependency.